src/main/java/minicraft/core/io/Localization.java

```diff
@@ -1,28 +1,43 @@
 package minicraft.core.io;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Locale;
-
-import org.jetbrains.annotations.NotNull;
-
 import minicraft.core.Game;
+import minicraft.saveload.Load;
 import minicraft.util.Logging;
-
+import org.jetbrains.annotations.NotNull;
 import org.json.JSONObject;
 import org.tinylog.Logger;
 
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
 public class Localization {
 
 	public static final Locale DEFAULT_LOCALE = Locale.US;
+	public static final Locale DEBUG_LOCALE = Locale.ROOT; // This locale is used for debugging;
 
 	private static final HashMap<Locale, HashSet<String>> knownUnlocalizedStrings = new HashMap<>();
 	private static final HashMap<String, String> localization = new HashMap<>();
+	private static final HashMap<String, String> fallbackLocalization = new HashMap<>();
```

Comment:
Merge localization and fallbackLocalization, and try to remove knownUnlocalizedStrings.